src/attributes/codegen.md

```diff
@@ -355,15 +355,16 @@ trait object whose methods are attributed.
 
 ## The `instruction_set` attribute
 
-The *`instruction_set` attribute* may be applied to a function to enable code generation for a specific
-instruction set supported by the target architecture. It uses the [_MetaListPath_] syntax and a path
-comprised of the architecture and instruction set to specify how to generate the code for
-architectures where a single program may utilize multiple instruction sets.
+On some CPU architectures it is possible to mix more than one instruction set into a single program.
+The `instruction_set` attribute lets you control which instruction set a particular function will be generated for.
```

Comment:
Can this be kept to something closer to the original? The reference uses a specific template for the definition of an attribute. That is, the first sentence starts with:

```
The *`foo` [attribute]* <explanation of what it does, and where it can be applied>
```

perhaps something like:

```suggestion
The *`instruction_set` [attribute]* may be applied to a function to control which instruction set the function will be generated for.
This allows mixing more than one instruction set in a single program on CPU architectures that support it.
```